R/app.R

```diff
@@ -1683,6 +1746,10 @@ AzimuthApp <- function(
     x = 'Azimuth.app.reference',
     default = 'https://seurat.nygenome.org/references/pbmc'
   ),
+  demodataset = getOption(
+    x = 'Azimuth.app.demodataset',
+    default = '~/azimuth_reference/pbmc3k_RAW_counts.h5ad'
```

Comment:
I suggest that the default for the demo dataset is NULL and the button doesn't exist if the option isn't specified as something other than NULL.